### PR TITLE
Google Authentication Functions

### PR DESCRIPTION
**Pull request by emieczkowski**
The functions needed for Google authentication were added to AppDelegate. I am currently working on finish the profile and then sign-up pages, which will have a Google sign-in button added to them that uses the output of these functions to determine name, email, id.

### CODE REVIEW

**Review by lsizemore8 — COMMENTED** [review]
Generally this looks good to me, but let's not merge this in until we connect backend & get the client ID in, it doesn't seem worthwhile to merge non-finished code to master. If we're finding that both of you need this on master to continue on, we'll reconsider.

File: carriage-user-ios/carriage-user-ios/AppDelegate.swift [hunk 1]
```diff
@@ -5,16 +5,26 @@
...
 
     var window: UIWindow?
 
 
     func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
-        // Override point for customization after application launch.
+        
+        // Initialize sign-in
+        GIDSignIn.sharedInstance().clientID = "YOUR_CLIENT_ID"
+        GIDSignIn.sharedInstance().delegate = self
```
Comment (lsizemore8) [inline carriage-user-ios/carriage-user-ios/AppDelegate.swift:20]: Add in GIDSignIn.sharedInstance().hostedDomain = "cornell.edu" so we can limit the sign in to cornell emails only :)

**Review by lsizemore8 — CHANGES_REQUESTED** [review]
My bad, did just notice this issue. Add the line I commented/make sure that doesn't break anything. If it doesn't, then I believe the only thing we're waiting on would be backend and us setting up a client ID

**Comment by emieczkowski** [comment]
WIP: still need client id from backend (currently has one I generated to prove that the functionality works)

**Pull request closed without merging**